test/parallel/test-http-client-abort2.js

```diff
@@ -30,7 +30,9 @@ const server = http.createServer(common.mustCall((req, res) => {
 server.listen(0, common.mustCall(() => {
   const options = { port: server.address().port };
   const req = http.get(options, common.mustCall((res) => {
+    res._dump = common.mustCall(res._dump.bind(res));
     res.on('data', (data) => {
+      req.abort();
```

Comment:
Does it make sense to make this a separate test rather than modifying the existing test? My concern is that the existing test is such a straightforward one and this kind of makes it less obvious what's going on. (There's no comment explaining why `req.abort()` is being called twice. But also wouldn't we want a test where it is only called once?)

Reply:
(Above is a question, not a blocker to landed.)